crates/api_models/src/refunds.rs

```diff
@@ -132,29 +135,20 @@ pub struct RefundListRequest {
     pub payment_id: Option<String>,
     /// Limit on the number of objects to return
     pub limit: Option<i64>,
-    /// The time at which refund is created
-    #[serde(default, with = "custom_serde::iso8601::option")]
-    pub created: Option<PrimitiveDateTime>,
-    /// Time less than the refund created time
-    #[serde(default, rename = "created.lt", with = "custom_serde::iso8601::option")]
-    pub created_lt: Option<PrimitiveDateTime>,
-    /// Time greater than the refund created time
-    #[serde(default, rename = "created.gt", with = "custom_serde::iso8601::option")]
-    pub created_gt: Option<PrimitiveDateTime>,
-    /// Time less than or equals to the refund created time
-    #[serde(
-        default,
-        rename = "created.lte",
-        with = "custom_serde::iso8601::option"
-    )]
-    pub created_lte: Option<PrimitiveDateTime>,
-    /// Time greater than or equals to the refund created time
-    #[serde(
-        default,
-        rename = "created.gte",
-        with = "custom_serde::iso8601::option"
-    )]
-    pub created_gte: Option<PrimitiveDateTime>,
+
+    pub time_range: Option<TimeRange>,
+    pub connector: Option<String>,
+    pub currency: Option<Currency>,
+    pub refund_status: Option<enums::RefundStatus>,
+}
+
+#[derive(Debug, Clone, Copy, serde::Serialize, serde::Deserialize, PartialEq, Eq, Hash)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Is there a need for this to be `camelCase`? As far as I know, almost everything on our API specification uses `snake_case`.